### DISCUSSION TITLE
Search overs the conversation messages

### DISCUSSION THREAD

**Discussion by martin-hoeger** [intellibar/main#47] [Ideas]
It would be super helpful if the search functionality also searched the conversation messages. Currently, it searches only the self-generated titles. It is quite difficult to find something...

**Comment by erusev** [comment]
I agree, we did something similar for our other app, [Nota](https://nota.md/) -- should be easy to do it it here as well.

**Comment by erusev** [comment]
> Currently, it searches only the self-generated titles. It is quite difficult to find something...

In case it's helpful, you can click on the title of the chat to set a custom title.
